### ISSUE TITLE
Redactor plugin not found: source

### ISSUE THREAD

**Issue by seandelaney** [craftcms/redactor#34]
New Craft 3 installation, v3.0.5.

From within the plugin store, clicked to install Redactor plugin, v 2.0.0.1.

I can add new Redactor fields but when I try to create or edit Entries, I get the following error:

```
Invalid Configuration – yii\base\InvalidConfigException
Redactor plugin not found: source

 in /Users/seandelaney/Sites/REMOVED/vendor/craftcms/redactor/src/Field.php at line 107
        $view->registerCssFile("{$baseUrl}/{$plugin}.css");
                    }
                    // Don't do this twice
                    self::$_registeredPlugins[$plugin] = true;
                    return;
                }
            }
        }
 
        throw new InvalidConfigException('Redactor plugin not found: '.$plugin);
    }
 
    /**
     * Returns the registered Redactor plugin paths.
     *
     * @return string[]
     */
    public static function redactorPluginPaths(): array
    {
                
2. in /Users/seandelaney/Sites/REMOVED/vendor/craftcms/redactor/src/Field.php at line 257 – craft\redactor\Field::registerRedactorPlugin('source')
        $redactorLang = $bundle->redactorLang ?? 'en';
 
        // register plugins
        $redactorConfig = $this->_getRedactorConfig();
        if (isset($redactorConfig['plugins'])) {
            foreach ($redactorConfig['plugins'] as $plugin) {
                static::registerRedactorPlugin($plugin);
            }
        }
 
        $id = $view->formatInputId($this->handle);
        $site = ($element ? $element->getSite() : Craft::$app->getSites()->getCurrentSite());
 
                
3. in /Users/seandelaney/Sites/REMOVED/vendor/twig/twig/lib/Twig/Extension/Core.php at line 1605 – craft\redactor\Field::getInputHtml(null, craft\elements\Entry)
```
I've not made any Redactor config changes.

**Comment by seandelaney** [comment]
Updated `Standard.json` config to resolve.

**Comment by brandonkelly** [comment]
Thanks, we fixed this in v2.0.1.

**Comment by kgrote** [comment]
In case anyone else is having this issue I fixed it by updating the permissions on my Redactor config files to 770. I also re-uploaded the config files just to be safe.
